### DISCUSSION TITLE
New progress icons

### DISCUSSION THREAD

**Discussion by PredragC** [freeplane/freeplane#291] [Show and tell]
Hi 
I made new progress icons with blue colors and cleaner. 
![image](https://user-images.githubusercontent.com/102694585/161329947-b1e6498b-a13b-41f0-9202-38a69482a68a.png)
What do you think, it is better to leave the old red color or use new blue color?

**Comment by rexelbartolome** [comment]
I think it's a cool update to let some of the other map icons be updated too and generally have this same feel as the menu icons :)
